var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -538,6 +541,10 @@ def configure_args(self):
         if spec.satisfies('+singularity @:4.9'):
             singularity_opt = '--with-singularity={0}'.format(spec['singularity'].prefix)
             config_args.append(singularity_opt)
+        # Lustre filesystem support
+	if spec.satisfies('+lustre'):
```

Comment:
This indentation looks odd...

Reply:
From branch it looks like this

```
        # Lustre filesystem support
        if spec.satisfies('+lustre'):
            lustre_opt = '--with-lustre={0}'.format(spec['lustre'].prefix)
            config_args.append(lustre_opt)
```

Reply:
Ah, that's because you have a tab in there:
```suggestion
        if spec.satisfies('+lustre'):
```

Reply:
Thanks, fixed.